DataFormats/Detectors/CTP/src/RunManager.cxx

```diff
@@ -134,6 +135,14 @@ int CTPRunManager::addScalers(uint32_t irun, std::time_t time)
       // LOG(info) << "Scaler for detector:" << countername << ":" << detcount;
     }
   }
+  */
+  int NINPS = 48;
+  int offset = 599;
+  for (uint32_t i = 0; i < NINPS; i++) {
+    uint32_t inpcount = mCounters[offset + i];
+    scalrec.scalersInps.push_back(inpcount);
```

Comment:
@shahor02 but here @lietava is adding some counters to the scalers, no?

Reply:
Ops, sorry, missed that part, perhaps was checking only particular commits. But then the ClassDef of CTPScalerRecordRaw and CTPScalerRecordO2 need to be incremented.

Reply:
Data members were not changed. I reused existing data member

Reply:
But data members are changed... `scalersDets` is removed and `scalersInps` is added!